cypress/integration/projectSettings/project_settings.ts

```diff
@@ -46,7 +46,7 @@ describe("Access page", () => {
       .first()
       .type("admin");
     cy.dataCy("save-settings-button").should("be.enabled").click();
-    cy.validateToast("success", "Successfully updated project");
+    cy.validateToast("success", "Successfully updated project", true);
```

Comment:
Nice work, this makes a lot of sense!

Did you consider updating the `validateToast` command to close by default? It seems like that could help avoid this issue in the future since it proved a bit mysterious.

Reply:
great suggestion! done! 😼